[2024-08-07T15:02:25.699+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T15:02:25.737+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-space-exploration.Respuesta_Satelite scheduled__2024-08-05T00:00:00+00:00 [queued]>
[2024-08-07T15:02:25.752+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-space-exploration.Respuesta_Satelite scheduled__2024-08-05T00:00:00+00:00 [queued]>
[2024-08-07T15:02:25.753+0000] {taskinstance.py:2306} INFO - Starting attempt 6 of 6
[2024-08-07T15:02:25.777+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Respuesta_Satelite> on 2024-08-05 00:00:00+00:00
[2024-08-07T15:02:25.784+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=329) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T15:02:25.785+0000] {standard_task_runner.py:64} INFO - Started process 334 to run task
[2024-08-07T15:02:25.786+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***-space-exploration', 'Respuesta_Satelite', 'scheduled__2024-08-05T00:00:00+00:00', '--job-id', '369', '--raw', '--subdir', 'DAGS_FOLDER/***-space-exploration.py', '--cfg-path', '/tmp/tmplrxrjhns']
[2024-08-07T15:02:25.788+0000] {standard_task_runner.py:91} INFO - Job 369: Subtask Respuesta_Satelite
[2024-08-07T15:02:25.857+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow-space-exploration.Respuesta_Satelite scheduled__2024-08-05T00:00:00+00:00 [running]> on host 38873943cc86
[2024-08-07T15:02:25.979+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-space-exploration' AIRFLOW_CTX_TASK_ID='Respuesta_Satelite' AIRFLOW_CTX_EXECUTION_DATE='2024-08-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-05T00:00:00+00:00'
[2024-08-07T15:02:25.981+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T15:02:27.052+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-07T15:02:27.053+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T15:02:27.065+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***-space-exploration, task_id=Respuesta_Satelite, run_id=scheduled__2024-08-05T00:00:00+00:00, execution_date=20240805T000000, start_date=20240807T150225, end_date=20240807T150227
[2024-08-07T15:02:27.130+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T15:02:27.182+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T15:02:27.186+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
